docs/cloud/cloud-org/cloud-org-billing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Manage billing
+layout: default
+parent: Manage organization
+grand_parent: Cloud
+nav_order: 5
+---
+
+# How do I manage billing for my organization?
+
+{: .no_toc }
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+* [Database usage pricing](https://www.featurebase.com/pricing){:target="_blank:}
+* [Available database sizes](https://docs.featurebase.com/docs/cloud/cloud-databases/cloud-db-shape/)
+* [Upgrade to paid](/docs/cloud/cloud-org/cloud-org-upgrade-to-paid/)
+
+## Billing cycle
+
+* Billing follows a 30-day cycle that starts on the date your organization was created.
+
+## Paying your bill
+* At the end of the billing cycle, an invoice is issued and the payment method you have on file will automatically be charged the listed amount
```

Comment:
An invoice is issued at the end of the billing cycle and your payment method is automatically charged.